src/nvim/tui/tui.c

```diff
@@ -508,6 +522,11 @@ static void sigwinch_cb(SignalWatcher *watcher, int signum, void *data)
   ui_schedule_refresh();
```

Comment:
we should change this to only schedule a refresh if the TUI size actually changed (could be done in a follow-up PR if it gets too messy)